### ISSUE TITLE
Predefined train/validation splits for SigOptSearchCV.

### ISSUE THREAD

**Issue by junwoony** [sigopt/sigopt-examples#154]
Hi Sigopt,
I'm trying to run Hyper Parameter Optimization using predefined(fixed) train/validation indices.

```
clf = SigOptSearchCV(net, net_parameters, cv=train_test_splitter, client_token=client_token,
                    n_jobs=1, n_iter=100, scoring=get_scorer('neg_mean_squared_error'),experiment=experiment)
```
This is what I used before with `train_test_splitter=ShuffleSplit() `from sklearn.
How can I modify SigOptSearchCV and optimize hyper parameters for the fixed train/validation splits?

I also tried PredefinedSplit from sklearn with fixed train and validaiton indices, but the cross validation generates separate test sets and divide train/val using the remaining sets.
The point is I don't want to leave test set out and report scores on validation set.
For example, If I have 1,000 datasets, I would like to use the first 800 for training ONLY, and the rest 200 for validation ONLY. 
But what the SigOptSearchCV did is, split the dataset into 900 for train and 100 for test. Then split the 900 data into train and validation sets.

Thank you!
